src/main/java/ch/ethz/idsc/retina/dev/rimo/RimoRateControllerWrap.java

```diff
@@ -15,21 +15,23 @@ public final void getEvent(RimoGetEvent rimoGetEvent) {
     this.rimoGetEvent = rimoGetEvent;
   }
 
-  /** @param pair vector of length == 2 with desired angular rates on the left and right rear wheel
+  /** @param rate_target desired average rate of left and right rear wheel in unit "rad*s^-1"
+   * @param angle of steering without unit but with interpretation in radians
    * @return */
-  public final Optional<RimoPutEvent> iterate(Scalar vel_target, Scalar theta) {
+  public final Optional<RimoPutEvent> iterate(Scalar rate_target, Scalar angle) {
     if (Objects.nonNull(rimoGetEvent))
+      // TODO DUBENDORF remove try below after testing on gokart
```

Comment:
Is it because try slows down the execution or there is a better reason? Anyway, I agree, I am just curious.

Reply:
this part is only executed at 50Hz so speed is not an issue. the implementations don't do anything out of the ordinary, thus i have removed the "try" in the latest commit altogether.